integration-tests/pages/reviewer/notCompletedIncidentsPage.js

```diff
@@ -0,0 +1,61 @@
+import page from '../page'
+import tabs from '../sections/incidentTabs'
+
+const row = (type, i) => cy.get(`[data-qa=${type}] tbody tr`).eq(i)
+
+const todoCol = (i, j) => row('incidents-todo', i).find('td').eq(j)
+
+const completeCol = (i, j) => row('incidents-complete', i).find('td').eq(j)
+
+const incidentsPage = () =>
+  page('Use of force incidents', {
+    ...tabs,
+    filter: {
+      prisonerName: () => cy.get('[name="prisonerName"]'),
+      prisonNumber: () => cy.get('[name="prisonNumber"]'),
+      reporter: () => cy.get('[name="reporter"]'),
+      dateFrom: () => cy.get('[name="dateFrom"]'),
+      dateTo: () => cy.get('[name="dateTo"]'),
+      apply: () => cy.get('[data-qa="apply"]'),
+      clear: () => cy.get('[data-qa="clear"]'),
+    },
+    getTodoRows: () => cy.get('[data-qa=incidents-todo]').find('tbody').find('tr'),
+    getNoTodoRows: () => cy.get('[data-qa=no-incidents-todo]'),
+    // getCompleteRows: () => cy.get('[data-qa=incidents-complete]').find('tbody').find('tr'),
+    // getNoCompleteRows: () => cy.get('[data-qa=no-incidents-complete]'),
+    getTodoRow: i => ({
+      row: () => cy.get('[data-qa=incidents-todo]'),
+      date: () => todoCol(i, 0),
+      prisoner: () => todoCol(i, 1),
+      prisonNumber: () => todoCol(i, 2),
+      reporter: () => todoCol(i, 3),
+      viewStatementsButton: () => todoCol(i, 4).find('a'),
+      overdue: () => cy.get('[data-qa=overdue]'),
+      reportId: () =>
+        todoCol(i, 4)
+          .find('a')
+          .invoke('attr', 'href')
+          .then(link => link.match(/\/(.*?)\/view-statements/)[1]),
+    }),
+
+    // getCompleteRow: i => ({
```

Comment:
need to get rid of all this commented code

Reply:
yes, i will do a clean up. Just kept it there for reference